react-app/src/App.js

```diff
@@ -0,0 +1,115 @@
+import './App.css';
+import React, {useState} from "react";
+
+import AddButton from "./addButton";
+import DeleteButton from "./deleteButton";
+import TaskList from './taskList';
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+import Alert from "./Alert";
+
+function App(props) {
+
+    const [todo, setTodo] = useState(props.initialData);
+
+    const [completedList, setCompletedList] = useState(todo.filter(item => item.isChecked));
```

Comment:
completedList and unCompletedList shouldn't be state variables. They should just be computed from todo.filter(...)